developer_guide/installation/installation_archive.rst

```diff
@@ -0,0 +1,123 @@
+Installing Akeneo PIM with the Archive
+======================================
+
+This document provides step by step instructions to install the PIM with the archive on an environment that fulfill the expected :doc:`system_requirements/system_requirements`.
+
+Getting Akeneo PIM
+------------------
+
+Downloading the archive
+***********************
+
+From http://www.akeneo.com/download you can choose to download a PIM edition with (called *icecat*) or without (called *minimal*) demo data. If you prefer, you can also download them directly from the command line:
+
+.. code-block:: bash
+    :linenos:
+
+    $ wget http://download.akeneo.com/pim-community-standard-v1.5-latest-icecat.tar.gz #for icecat version
```

Comment:
Also it's an alternative. The first sentence tells to download the archive from the site.